### DISCUSSION TITLE
How does the REPORT CAPTCHA block work?

### DISCUSSION THREAD

**Discussion by Aholicknight** [openbullet/openbullet#856] [Q&A] [answered]
![silverbullet_0264478706](https://user-images.githubusercontent.com/7843719/184031719-8f0121eb-5c80-4715-8347-2ddfec207fc3.png)

I am trying to use the report captcha block and it says `An error occurred! Input string was not in a correct format.` everytime I try to use it. I can't find any documentation on how to use this, would I replace the Captcha ID with `g-recaptcha` or what would I do?

Thank you!

**Comment by openbullet** [comment, marked as answer]
OB1 is out of date, please switch to OB2

**Comment by Aholicknight** [comment]
Will do
